app/views/hosts/_templates.html.erb

```diff
@@ -0,0 +1,38 @@
+<% if @host.managed? %>
```

Comment:
`SETTINGS[:unattended]` is unnecessary here, see my [comment](https://github.com/theforeman/foreman/pull/3376#discussion_r60204301) below